### ISSUE TITLE
Close tab button is on the wrong side on macOS

### ISSUE THREAD

**Issue by kitgrose** [warpdotdev/Warp#1847]
### Describe the bug

In macOS, window and tab controls are conventionally on the left side (as opposed to Windows, where controls are normally on the right).

Warp shows the close button for each tab on the _right_ of the tab, which breaks the convention on the platform (see Safari, Finder, Terminal, etc. for standard tab controls on the system, all of which have their close buttons on the left).

### To Reproduce

1. Open multiple tabs in Warp
2. Hover over a tab with the mouse
3. Observe that the close button is on the right of the tab

### Expected behaviour

The close button should be on the left of the tab

### Screenshots

<img width="247" alt="Screen Shot 2022-09-19 at 11 39 41 pm" src="https://user-images.githubusercontent.com/165977/191030706-9e6975b0-740a-4a56-91f5-8c09352ae80e.png">

### Operating System

MacOS

### OS Version

12.5

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin21.0)

### Warp Version

v0.2022.09.13.08.15.stable_03

### Does this block you from using Warp daily?

No

**Comment by Subterrane** [comment]
This is interesting. I see in Terminal, and Apple app, the close button is on the left side of the tab. But in apps that tend to be cross-platform (Chrome, Firefox, Visual Studio Code) the close button is on the right.

**Comment by dannyneira** [comment]
Hi @kitgrose, I'm marking this as a feature request because @Subterrane is correct. 

Native apps like Safari close tabs on the left, while cross-platform apps ( which is our intention to eventually support Linux/Windows/Web ), close on the right side. We will post any updates to this request here.

**Comment by kitgrose** [comment]
I understand this decision, although it's worth pointing out that there are absolutely cross-platform apps that don't behave this way (e.g. VS Code, Adobe apps, etc.).

**Comment by partydrone** [comment]
@kitgrose is correct that there are cross-platform apps that place the close button on the left. May I suggest, @dannyneira, that this be a setting? Here you can see in my VS Code setup, that the close button is on the left based on a setting:

<img width="725" alt="Screenshot 2023-01-24 at 8 29 01 AM" src="https://user-images.githubusercontent.com/57892/214321432-ca6a12d5-be9b-46b1-8a41-61f1c08b4e16.png">

<img width="447" alt="Screenshot 2023-01-24 at 8 25 13 AM" src="https://user-images.githubusercontent.com/57892/214321474-c37c8cf2-e303-4201-83f6-823812b8b5f7.png">

**Comment by kpmcguire** [comment]
Came here to file this issue, pleasantly surprised to see it has already been filed. 

For the record, Jetbrains products are cross-platform and also have a setting for this. 

This is one of those attention-to-detail things that I think would go over well with certain old skool mac enthusiasts.

**Comment by narcistesa** [comment]
Agree, this is a Mac-only app right now and it should follow macOS ui standards.

**Comment by ttuygun** [comment]
Is there any progress on this?

**Comment by dannyneira** [comment]
No progress or eta at this time. To realistically set expectations, we focus on the requests with the most 👍 's so the more we can get on this request the better chance it'll be prioritized. 

As a stopgap measure, you can quickly close the current session with `CMD-W` whether it's the active window, tab, or pane. Appreciate your patience and understanding!

**Comment by kpmcguire** [comment]
> you can quickly close the current session with CMD-W

This has nothing to do with the issue whatsoever.

**Comment by dannyneira** [comment]
Not my intention really, I understand it's more of a UI/UX issue. Just trying to provide an alternative for closing the tabs since the current implementation is not up to modern standards (i.e. configurable/left-side tab close position).

**Comment by Z2Flow** [comment]
What separates great from merely good is attention to detail.
What I find intolerable about Windows is the pervasive "Good enough" attitude.

**Comment by Z2Flow** [comment]
We now have "great" new icons.
This is definitely a higher priority than providing the tab close button on the left.

**Comment by dannyneira** [comment]
Hey folks, You'll have the option to move the Tab close button to the left side in today's release. https://docs.warp.dev/terminal/appearance/tabs-behavior#tab-close-button

![Image](https://github.com/user-attachments/assets/42c66898-d197-44fb-8687-765601fd8a12)

Closing as resolved, but please let us know if any issues.

**Comment by Z2Flow** [comment]
At last. Thanks!
